packages/flutter_tools/gradle/flutter.gradle.kts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+apply<FlutterPluginKts>()
```

Comment:
Do we already require of all users the dependencies needed to use Kotlin during builds?

Reply:
I think so but let me double check that.

Reply:
https://docs.gradle.org/5.0/release-notes.html

Kotlin DSL 1.0 is supported in Gradle 5.0 (Nov 2018), and older versions of gradle include pre-1.0 support for Kotlin DSL. This predates Dec 2018 Flutter 1.0 release, so all flutter apps 1.0 and above should support Kotlin DLS buildscripts.

Gradle 3.2 (Nov 2016) contains the initial support for Kotlin DSL (called Gradle Buildscript Kotlin) back then.

Reply:
For reference, Flutter 1.0 shipped with Gradle 4.10.2, which included Kotlin DSL 1.0 RC6 https://docs.gradle.org/4.10.2/release-notes.html

Reply:
Cool, sounds like Kotlin DSL ships with an embedded compiler: https://docs.gradle.org/current/userguide/migrating_from_groovy_to_kotlin_dsl.html